extension/src/experiments/model/filterBy/quickPick.ts

```diff
@@ -67,11 +69,43 @@ export const operators = [
     label: Operator.NOT_CONTAINS,
     types: ['string'],
     value: Operator.NOT_CONTAINS
+  },
+  {
+    description: 'After Date',
+    label: Operator.AFTER_DATE,
+    types: [ColumnType.TIMESTAMP],
+    value: Operator.AFTER_DATE
+  },
+  {
+    description: 'Before Date',
+    label: Operator.BEFORE_DATE,
+    types: [ColumnType.TIMESTAMP],
+    value: Operator.BEFORE_DATE
+  },
+  {
+    description: 'On Day',
+    label: Operator.ON_DATE,
+    types: [ColumnType.TIMESTAMP],
+    value: Operator.ON_DATE
   }
 ]
 
+const getValue = (operator: Operator): Thenable<string | undefined> => {
+  if (isDateOperator(operator)) {
+    return getValidInput(
+      Title.ENTER_FILTER_VALUE,
+      (text?: string): null | string =>
+        isFreeTextDate(text)
+          ? null
+          : 'please enter a valid date of the form yyyy-mm-dd',
+      getIsoDate()
+    )
+  }
+  return getInput(Title.ENTER_FILTER_VALUE)
+}
+
```

Comment:
Also, probably minor but due to the the "Created" text being left aligned on the column, the filter covers up the text no matter how large the column is: 

<img width="230" alt="image" src="https://user-images.githubusercontent.com/43496356/187434900-b0ceb570-6aa2-46ea-9170-515f70da5e09.png">

Reply:
Right side for the left-aligned content?